### DISCUSSION TITLE
show preview

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [maruohon/litematica#869] [Q&A]
how to show preview in the load schematic screen

**Comment by maruohon** [comment]
You can set the preview thumbnail via the Schematic Manager menu. Select the schematic, click on Set Preview, then exit the menu. You will have a white helper frame to line up the view. Hit F1 to hide the HUD, then hit F2 to take the screenshot, and it will be set as the preview image, and then F1 again to show the HUD again. If you want to cancel the preview task, right click the Set Preview button.
